backend-rust/README.md

```diff
@@ -0,0 +1,59 @@
+# CCDScan Backend
+
+This is the backend for [CCDScan](https://ccdscan.io/) Blockchain explorer for the [Concordium blockchain](https://concordium.com/).
+
+The backend consists of two binaries:
+
+- `ccdscan-indexer`: Traversing the chain indexing events into the database.
+- `ccdscan-api`: Running a GraphQL API for querying the database.
+
+The service is split to allow for running several instances of the GraphQL API and while having a single instance of the indexer.
+
+## Dependencies
+
+To run the service, the following dependencies are required to be available on the system:
+
+- PostgreSQL server 16
+
+## Setup for development
+
+To develop this service the following tools are required, besides the dependencies listed above:
+
+- [Rust and cargo](https://rustup.rs/)
+- [sqlx-cli](https://crates.io/crates/sqlx-cli)
+
+Then set the environment variable `DATABASE_URL` pointing to the location of the SQL database, this can be done by creating a `.env` file within this directory.
```

Comment:
This could be avoided using sqlx's offline feature, but it will still be necessary when adding or editing queries.

Reply:
Sure, but contributors should not rely on that